config/config.go

```diff
@@ -89,6 +90,7 @@ func newDefaultConfig() *viper.Viper {
 	c.SetDefault("min", 1)
 	c.SetDefault("max", math.MaxInt64)
 	c.SetDefault("remove-empty", true)
+	c.SetDefault("remove-focused", true)
```

Comment:
@cmschuetz Should `btops` default to not removing focused desktops by default (`false`) or should it keep the previous behavior of removing focused desktops (`true`)?

Reply:
I would prefer to keep it in line with previous behavior, so I think `true` here works.

With that said, `remove-focused` is sort of tied to `remove-empty` as it shouldn't have any effect if `remove-empty` is false and so this naming might be a bit confusing taken out of context

Was thinking about this a bit and figured we could merge this config with `remove-empty` but instead provide a distinct option for this case.  

E.g. possible options: `remove-empty: [true, unfocused, false]`

We'd have to change this config from a bool to a string but I think this strikes a nice balance without adding more top-level configs and would not be a breaking change.

Long term I was thinking we could possibly supply a white-list of applications where this rule applies.  E.g. remove all empty, focused desktops except when the last closed program was steam.  This is something I would personally want as it's annoying to open steam while it brings up and tears down several windows.  However, this would require a substantial code change as we'd need a log of previous bspwm states and I think we can solve this problem in other ways, will create issues for some more ideas

Reply:
That makes sense, and I agree (looking back now) that remove-focused is confusing.

Perhaps desktop pinning would be a solution to your second note?
I.E. allow a way to "pin" a desktop from being removed even `remove-empty = true`.

For a whitelist, maybe a configurable timeout before removing could be written? For example, if Steam opens on a desktop, that desktop will have a timeout of say 60s before it is removed if it's empty. This can rely on a check added to the main loop.